src/intl/en.json

```diff
@@ -800,6 +800,15 @@
         "delegated": "Delegated {vp} VP",
         "delegate_action": "Delegate them your VP",
         "button": "Show all delegators"
+      },
+      "voted_proposals": {
+        "title": "Proposals voted on",
+        "helper": "...",
```

Comment:
This helper is missing right? Maybe we should add a `// TODO:`

Reply:
No helper text according to Gino. Deleted